tests/conditional_compilation/test_verify.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+""" Test to verify inference results.
+"""
+
+import numpy as np
+from tests_utils import run_infer
+
+
+def test_verify(test_id, model, artifacts, openvino_root_dir, tolerance=1e-6):  # pylint: disable=too-many-arguments
+    """ Test verifying that inference results are equal
+    """
+    out = artifacts / test_id
+    install_prefix = artifacts / test_id / "install_pkg"
+    returncode, output = run_infer(model, f"{out}.npz", openvino_root_dir)
```

Comment:
please, define variables for paths to prevent hardcoding in several places (e.g. 17 and 21 lines)